packages/web/server/queries/complex/assets/index.ts

```diff
@@ -36,8 +38,18 @@ export async function getAsset({
   assetList?: AssetList[];
   anyDenom: string;
 }): Promise<Asset | undefined> {
-  const assets = await getAssets({ assetList, findMinDenomOrSymbol: anyDenom });
-  return assets[0];
+  return cachified({
+    cache: getAssetCache,
+    key: anyDenom,
+    getFreshValue: async () => {
+      const assets = await getAssets({
+        assetList,
+        findMinDenomOrSymbol: anyDenom,
+      });
+      return assets[0];
+    },
+    ttl: 10 * 1000, // 10 seconds
+  });
```

Comment:
Is this just to save on the array access? I think `getAssets` is cached.

Reply:
You're right. Removed this cachified